### PR TITLE
Completed UI

### PR DESCRIPTION
**Pull request by PeithonKing**

### CODE REVIEW

File: src/static/script_game.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	check = check_input(inputValue);
+
+	if (check) {
+		guessedletters.push(inputValue);  // Add the entered letter to the guessedletters array
+		console.log(guessedletters);
+		fetch("/game/" + uid)  // fetch the game data
+			.then(response => response.json())
+			.then(data => {
+				/*
+				I forgot exactly how we decided the API response should be... so I assume it's like this:
```
Comment (dhananjaylatkar) [inline src/static/script_game.js:23]: Can we confirm and finalize this?

File: src/static/script_game.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+				// Update the imageelement's src attribute
+				imageelement.src = image;
+			})
+			.catch(error => {
+				console.error("Error fetching game data:", error);
+			});
+	}
+}
+
+function check_input(inputValue) {
```
Comment (dhananjaylatkar) [inline src/static/script_game.js:51]: i guess this info will come from game logic. do we need it here?

File: src/static/script_game.js (hunk 1 above)
Comment (PeithonKing) [reply src/static/script_game.js:23]: You do your part... complete your part, I will check my part, adapt to your code after that...

File: src/static/script_game.js (hunk 2 above)
Comment (PeithonKing) [reply src/static/script_game.js:51]: Yes, I explained before. Minimising requests to the server is a target. We don't want to waste requests just like that. Whatever can be done in the front end would be done in the front end.

File: src/static/script_game.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+let lifesign = "❤️";
+let uid = document.getElementById("uid").innerText;
+// console.log(`uid: ${uid}`);
+let liveselement = document.getElementById("lives");
+let imageelement = document.getElementById("hint");
+let wordtable = document.getElementById("wordtable");
+let warning = document.getElementById("warning");
+let guessedletters = [];
+
+function foo(inputValue) {
```
Comment (dhananjaylatkar) [inline src/static/script_game.js]: change this func name

Comment (PeithonKing) [reply src/static/script_game.js]: I will... sorry!

**Pull request merged**